plugins/mta/src/components/ApplicationInventory  /ApplicationInventory.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import useAsync from 'react-use/lib/useAsync';
+
+import { Table } from '@backstage/core-components';
+import { configApiRef, useApi } from '@backstage/core-plugin-api';
+
+import { Box, Grid, Typography } from '@material-ui/core';
+
+import { Columns, Data } from './tableHeading';
+
+export const ApplicationInventory = () => {
+  const config = useApi(configApiRef);
+  const BACKEND_URL = 'backend.baseUrl';
+
+  const { loading: isSysInfoLoading, value: sysInfoData } =
+    useAsync(async (): Promise<Data> => {
+      const backendUrl = config.getString(BACKEND_URL);
+      // TODO: Update the URL to match the new MTA client's backend .
+      const backendApiEndPoint = `${backendUrl}/hub/applications`;
```

Comment:
If you want to have a configurable way to change this endpoint, it can be added to the `app-config.yaml` file. Let me know if you want to know more about this.

I'll approve the PR though :-)

Reply:
thanks, yup that's our next step to design the MTA client and add the configuration app-config.yaml
cc @tmihalac